tests/wicked/basic/sut/t05_dynamic_addresses_xml.pm

```diff
@@ -24,7 +24,6 @@ sub run {
     record_info('Info', 'Set up dynamic addresses from wicked XML files');
     $self->get_from_data('wicked/dynamic_address/dynamic-addresses.xml', "/data/dynamic_address/dynamic-addresses.xml");
     assert_script_run("sed -i 's/xxx/$iface/g' /data/dynamic_address/dynamic-addresses.xml");
-    assert_script_run("ifdown $iface");
     $self->wicked_command('ifup --ifconfig /data/dynamic_address/dynamic-addresses.xml', $iface);
     $self->assert_wicked_state(ping_ip => '10.0.2.2', iface => $iface);
```

Comment:
Should we check, if the current IP is different to the static one? As mentioned in [wicked_1_basic.feature](https://github.com/openSUSE/wicked-testsuite/blob/master/features/wicked_1_basic.feature#L67).

The IP seems to be checked in wicked testsuite more often. Is our implicit check for pinging the remote enough?

Reply:
you right , but I would treat it as improvement for future not in terms of this PR

Reply:
I was just thinking about it, cause we remove the ifdown and I'm not sure if the ifup on the running interface really do what we wont.
But sure we can do it  in a separate PR.

Reply:
what if the dynamic IP is the same as the static by coincidence?

Reply:
As we configure the DHCP server, lets make sure it isn't.

Reply:
That's a good point, but don't want to complicate this PR with another topic. I take the point and will create a ticket to do so, ok?

Reply:
ok